tools/generate-proxies/generate-angular-proxies.ts

```diff
@@ -163,7 +164,10 @@ async function generateComponentSource(element: LitCustomElement) {
 
       ${getInputProperties(element)}
       ${getOutputProperties(element)}
-    }`;
+    }
+
+    export { ${angularComponentName} as ${aliasName} };
+    `;
```

Comment:
Nice! I think the solution is a bit more clean if we add this to the generated index.ts file instead of here. 

Then it will be easier to manage and also remove at a later time.